api/control.proto

```diff
@@ -191,6 +191,8 @@ message ListTasksRequest {
 		repeated docker.swarmkit.v1.TaskState desired_states = 6;
 		// NamePrefixes matches all objects with the given prefixes
 		repeated string name_prefixes = 7;
+		repeated string runtimes = 9;
```

Comment:
The problem is it works for built-in types (Container, NetworkAttachment) but not for Generic.

For Generic, we index by `GenericRuntime.Kind`.

Basically, we assume runtimes are indexed by a string. Container and NetworkAttachment are special cases.

Makes sense?